fcrepo-indexing-http/src/main/java/org/fcrepo/camel/indexing/http/HttpRouter.java

```diff
@@ -35,7 +39,7 @@
  */
 public class HttpRouter extends RouteBuilder {
 
-    private static final Logger logger = getLogger(HttpRouter.class);
+    private static final Logger LOGGER = getLogger(HttpRouter.class);
```

Comment:
Out of curiosity, why did this change to ALL CAPS? (The equivalent logger in the SolrRouter uses lowercase naming).